x/concentrated-liquidity/swaps_test.go

```diff
@@ -1268,10 +1716,10 @@ func (s *KeeperTestSuite) TestCalcAndSwapInAmtGivenOut() {
 				s.Require().NoError(err)
 
 				// check that tokenIn, tokenOut, tick, and sqrtPrice from CalcOut are all what we expected
+				s.Require().Equal(test.expectedSqrtPrice, sqrtPrice)
```

Comment:
Out of curiosity, what prompted moving this check higher up?

Reply:
Sqrt price is calculated first. If it si wrong, the following values would be incorrect as well.